src/game.cpp

```diff
@@ -11778,6 +11778,12 @@ void game::vertical_move( int movez, bool force )
         m.ter_set( stairs, t_manhole );
     }
 
+    // Wouldn't work and may do strange things
+    if( u.is_hauling() && !m.has_zlevels() ) {
+        add_msg( _("You must enable experimental z-levels to haul across z-levels") );
```

Comment:
```suggestion
        add_msg( _( "You must enable experimental z-levels to haul across z-levels." ) );
```

Reply:
That's what I get for forgetting to run astyle. Thanks